components/support/nimbus-fml/scripts/build-dist.sh

```diff
@@ -0,0 +1,68 @@
+#!/usr/bin/env bash
+# This Source Code Form is subject to the terms of the Mozilla Public
+# License, v. 2.0. If a copy of the MPL was not distributed with this
+# file, You can obtain one at http://mozilla.org/MPL/2.0/.
+
+# Small script to be run on the CI server to cross compile nimbus-fml on Android
+# and iOS developer machines.
+#
+# It installs the tools and targets needed via brew and rustup, cross compiles the nimbus-fml
+# and then zips up all OS/architecture versions into a single archive.
+#
+# CircleCI then stores this as an artifact, and pushes it to Github on each release.
+#
+# This will be downloaded and unzipped as part of the buid processes for iOS and Android.
+targets="aarch64-apple-darwin x86_64-unknown-linux-musl x86_64-apple-darwin"
+dry_run=false
+
+# Assume we're in $root_dir/components/support/nimbus-fml/scripts
+root_dir=$(dirname "$0")/../../../..
+fml_dir=$root_dir/components/support/nimbus-fml
+target_dir=$root_dir/target
+filename=$(basename "$fml_dir")
+
+# But we'll dump the zip file wherever we run the script from.
+dist_file=${filename}.zip
+
+prompt='$'
+echo "## Installing tools for cross compiling"
+install_musl_cross="brew install filosottile/musl-cross/musl-cross"
```

Comment:
Can you write a comment on why we need this?

I see `x86_64-unknown-linux-musl` in our targets, so I imagine the target requires some native libraries that this provides? And for my own curiosity, why do we need to target `x86_64-unknown-linux-musl`, is it for devs running Linux machines?

Reply:
We have devs running linux, and Android CI builds are (IIRC) all done on linux.

Reply:
Aaah the CI builds of course 💯

Reply:
I've added a comment to explain all.